filesystems/cifs/connectathon/metadata

```diff
@@ -0,0 +1,14 @@
+[General]
+name=filesystems/cifs/connectathon
+owner=Jianhong Yin <jiyin@redhat.com>
+description=Connectathon for CIFS
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=wget;git;samba;time;patch;cups;samba-client;cifs-utils;gcc;bzip2;nfs-utils;net-tools;rusers
```

Comment:
What happens with dependencies like `python3-lxml`?

Reply:
Fixing another dependencies for filesystem tests.  :-)